packages/database/src/migrate.ts

```
@@ -0,0 +1,147 @@
+}
+
+// Main execution
+const command = process.argv[2];
+
+switch (command) {
+  case 'list':
+    await applyMigrations();
+    break;
+  case 'validate':
+    const isValid = validateMigrations();
+    process.exit(isValid ? 0 : 1);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)